libexec/scoop-reset.ps1

```diff
@@ -69,7 +69,8 @@ $apps | ForEach-Object {
 
     #region Workaround for #2952
     $processdir = $dir | Select-Object -ExpandProperty Path
-    if (Get-Process | Where-Object { $_.Path -like "$processdir\*" }) {
+    if ((Get-Process | Where-Object { $_.Path -like "$processdir\*" }) -and
+        (Read-Host "Application '$app' is still running. Do you want to proceed uninstallation? All unsaved data will be lost [y/N]") -ne 'y') {
```

Comment:
```suggestion
        (Read-Host "Application '$app' is still running. Do you want to proceed with the reset? All unsaved data will be lost [y/N]") -ne 'y') {
```
Not sure if this prompt is necessary, will any data actually be lost - doesn't reset just recreate shims and shortcuts?

Reply:
The "data loss" I meant is in-memory progress, like if you are editing a file without saving to disk. Nothing already on disk is not lost.

What's your suggestion?

Reply:
I don't think scoop stops the process when resetting. Have a look at lines 78-90 of `scoop-reset.ps1`, it's just recreating shims, start menu shortcuts, re-adding environment variables and any directories to the PATH variable, plus recreating the junctions for the `current` folder and the persisted folders. You can try for yourself, I just reset audacity while it was running and it wasn't stopped. It still remembers my preferences so persisted data was not lost either.

So if that's true my suggestion is remove this prompt for reset but keep it for update/uninstallation. (see my other suggestions below for better wording in update and uninstallation)

Reply:
Oh, you are talking specifically for the "reset" case. Sorry I misunderstood.

So the reason why you don't see the "stop process" message and the new prompt, I think, is because the current `scoop-reset.ps1` has a problem. Instead of getting the `$processdir = appdir $app $global | Resolve-Path | Select-Object -ExpandProperty Path` like `scoop-uninstall.ps1`, it's doing `$processdir = $dir | Select-Object -ExpandProperty Path`. This means the variable points to the versioned directory instead of the `current` junction. Since when you launch an app the current directory is most likely the `current` junction, that process running check in `scoop-reset.ps1` never worked. I just noticed this bug.

Once you change line 71 to the same as `scoop-uninstall.ps1`, you will see the prompt. Can you confirm?

Reply:
I had a look at #4310, yes it looks like this prompt is necessary. 

>Once you change line 71 to the same as scoop-uninstall.ps1, you will see the prompt. Can you confirm?

Yes I see the prompt.

In that case, my only suggestion for this file is change the wording from 'proceed uninstallation?' to 'proceed with the reset?'

Reply:
Yes. I'll change the wording if this approach is approved.